README-ja.md

```diff
@@ -30,18 +30,19 @@ $ npm i xpring-js
 サーバー側コンポーネントは、クライアント側のリクエストをXRPノードに送信します。
 
 開発者がすぐに開発を開始できるように, Xpringは現在、サーバー側コンポーネントをホスト型サービスとして提供し、クライアント側ライブラリからのリクエストをXRPノードにプロキシします。開発者は次のエンドポイントに到達できます:
+
 ```
-# TestNet - Node
-alpha.test.xrp.xpring.io:50051
+# TestNet - Node Environment
```

Comment:
nit: I think our capitalization is now `Testnet` (and `Mainnet`)

Reply:
https://app.asana.com/0/1143722888069470/1166110587347851

I pinky promise I will do this tomorrow; currently late in the day and I want to refactor all at once.

Reply:
(This change also unbreaks tests - hence urgency to want to land)